spec/common/fixup.js

```diff
@@ -46,7 +46,7 @@ function unComment(utils, content) {
 // If content is a self-citation, replace it with the document name
 function noSelfCite(utils, content) {
   if (content.toUpperCase() === `[[[${respecConfig.shortName}]]]`.toUpperCase()) {
-    return respecConfig.title + ' (this document)';
+    return respecConfig.title + '(this document)';
```

Comment:
Is the space removal intended (I have not checked anything, it's just to make sure it's not an unintended mistake)?

Reply:
Out-of-date change. Reversed.